### PR TITLE
Byteplanes

### PR DESCRIPTION
**Pull request by beppo-sturmtief**
Hi Lode,
as discussed via email, i've adopted the code to be able to work only with the single byte plane. Sorryfully this (of course) leads to a reduction in compression efficiency, but it seems moderate.

Kind regards,
Simon

### CODE REVIEW

File: fusion_power_video.cc [hunk 1]
```diff
@@ -106,10 +108,10 @@ chunk flags meanings:
  for all frames.
 
 image flags meanings:
--flags & 1: iff true, delta frame prediction is enabled. This must be false if
+-flags & 1: if true, delta frame prediction is enabled. This must be false if
```
Comment (lvandeve) [inline fusion_power_video.cc:111]: iff was not a typo but "if and only if". Nothing against the change though.

File: fusion_power_video.cc [hunk 2]
```diff
@@ -239,29 +245,24 @@ bool BrotliDecompress(const uint8_t* in, size_t size, size_t* pos,
   return true;
 }
 
-// Returns average entropy per symbol (a guess of bits per pixel).
+template<typename T> T approxLog2(T v) {
+   return ((unsigned) (8*sizeof(T) - __builtin_clzll((v)) - 1));
```
Comment (lvandeve) [inline fusion_power_video.cc:249]: __builtin_clzll is not available in all compilers, a fallback and guarding it with #if would be useful

**Comment by lvandeve** [comment]
Thanks, I'll merge this in. Please check a few comments I left

File: fusion_power_video.cc (hunk 1 above)
Comment (beppo-sturmtief) [reply fusion_power_video.cc:111]: Oh... I did not know that idiom... Learning something new each day - thanks!

**Pull request merged**